mappings/net/minecraft/entity/mob/EntityMob.mapping

```diff
@@ -1,6 +1,9 @@
 CLASS none/sq net/minecraft/entity/mob/EntityMob
+	CLASS none/sq$a
```

Comment:
Added by Enigma when I attempted to rename it but I reverted it back to default.